dash/backend/src/modules/command-line/services/cli-command-builder.service.ts

```diff
@@ -61,6 +61,12 @@ export class CliCommandBuilderService {
                 'Accepts comma delimited list of namespaces to whitelist',
                 () => this.helmSetupCommand.loadDefaultNamespaceExceptions(),
             ),
+            new CliCommand(
```

Comment:
Why do we need a CLI command for this? We have a UI for creating API Keys.

Reply:
Eh, fair enough. Though we could ask the same question of the runSeed() job.